script.js

```diff
@@ -1,5 +1,203 @@
-function runProgram(){
+function displayPresInformation(president){
+    console.log(president);
 
+    let CardContainer = document.querySelector(".bottom-container")
+    let newCard = createCard(president);
+    CardContainer.appendChild(newCard);
+
+}
+
+function displayPresName(president){
+    console.log("function displayPresName running");
+    let NameContainer = document.querySelector(".bottom-container");
+    let NameCard = createNameCard(president);
+    NameContainer.appendChild(NameCard);
+}
+
+function createNameCard(president){
+    let nameCard = document.createElement('p');
+    nameCard.classList.add('.info-container');
+
+    let PresName = document.querySelector('div');
+    PresName.classList.add('card-body');
+
+    let name = document.createElement('p');
+    name.textContent = `Name: ${president.name}`;
+    name.classList('card-title');
+    
+    PresName.appendChild(name);
+    nameCard.appendChild(nameCard);
+    
+    return PresCard;
+}
```

Comment:
You don't need to have a createNameCard etc funciton, since all of these functions would be similar.